_layouts/plain.html

```diff
@@ -5,7 +5,6 @@
 <div class="fill">
   <header>
     <nav class="page-centered">
-      <a class="vl-title" href="{{site.baseurl}}/">{{site.title}}</a>
```

Comment:
Vega didn't remove this and we wouldn't have a way to get back to the home page.